### PR TITLE
feat(#1471): Unplace Dependencies

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Set "unplace=true" for dependencies that aren't used by any other binary.

Closes: #1471

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Please, have a look.

File: eo-maven-plugin/src/main/java/org/eolang/maven/UnplaceMojo.java [hunk 1]
```diff
@@ -82,78 +84,90 @@ public void exec() throws IOException {
...
-            this.placeThem();
+            this.unplaceBinaries();
+            this.unplaceDependencies();
         }
     }
 
+    /**
+     * Mark dependencies as unplaced if all related binaries are unplaced.
+     */
+    private void unplaceDependencies() {
```
Comment (mximp) [inline eo-maven-plugin/src/main/java/org/eolang/maven/UnplaceMojo.java:95]: @volodya-lombrozo Maybe we can rename `dependencies`/`binaries` into `jars`/`classes`? It's quite hard to guess what is 'dependency' and what is 'binary' in this context.

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/main/java/org/eolang/maven/UnplaceMojo.java]: @mximp Done, please, have a look

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**